docs/source/reference/envvars.rst

```diff
@@ -311,6 +311,38 @@ Compilation options
 
     *Default value:* 128
 
+.. envvar:: NUMBA_EXPERIMENTAL_REFPRUNE_PASS
+
+    Turns on the LLVM pass level reference-count pruning pass and disables the
+    regex based implementation in Numba.
+
+    *Default value:* 1 (On)
+
+.. envvar:: NUMBA_EXPERIMENTAL_REFPRUNE_FLAGS
```

Comment:
Given it's on by default if this PR is merged as is, should it be called "experimental"?

Reply:
how about `NUMBA_LLVM_REFPRUNE_FLAGS`?

Reply:
> how about `NUMBA_LLVM_REFPRUNE_FLAGS`?

sounds good.

Reply:
renamed